code/modules/mob/living/basic/space_fauna/netherworld/creature.dm

```diff
@@ -21,7 +21,10 @@
 	unsuitable_atmos_damage = 0
 	unsuitable_cold_damage = 0
 	unsuitable_heat_damage = 0
-	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	// Real blue, trying to go for the migo's look
```

Comment:
Ditto on this not being the migo. (whatever the hell the "creature" is)

Reply:
see above